### PR TITLE
Leadership page styles

### PR DESCRIPTION
**Pull request by VladimirZaets**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix hlxsites/merative#103 

Test URLs:
- Before: https://main--merative2--hlxsites.hlx.page/drafts/vzaets/leadership
- After: https://leadership-page--merative2--hlxsites.hlx.page/drafts/vzaets/leadership

### CODE REVIEW

**Comment by amol-anand** [comment]
@VladimirZaets Desktop looks good. Tablet and Mobile CSS needs some tweaks.
Figma: https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?node-id=9144%3A42215&t=hiq19MO7XH7KXVBr-0

Tablet - the semi circle image doesn't fit right
Screenshot 2023-01-27 at 10 48 10 AM

Mobile - the semi circle image is too small for the width
Screenshot 2023-01-27 at 10 50 42 AM

I would look at the figma and see what tweaks are needed for tablet and mobile and then you should be good to go. minor stuff left.

**Comment by amol-anand** [comment]
In the CSS, start with mobile as the default and then put in any changes you need for tablet / desktop in their own media queries.

**Review by amol-anand — CHANGES_REQUESTED** [review]
In the CSS, start with mobile as the default and then put in any changes you need for tablet / desktop in their own media queries.

**Comment by VladimirZaets** [comment]
@amol-anand fixed, please take a look one more time, thanks

File: styles/styles.css [hunk 1]
```diff
@@ -614,3 +614,79 @@ main .divider{
...
+}
+main .section.leadership-content .columns>div>div>p {
+  font-style: normal;
+  font-size: 28px;
+  line-height: 120%;
+  letter-spacing: 0.0175em;
+}
+
+main .section.leadership-content .columns>div>div>h6 {
+  font-family: 'Alliance No.1';
```
Comment (amol-anand) [inline styles/styles.css:660]: Everything else looks good. Just the fonts are weird. 
you can update this line to use the global variable for this font like below:

    font-family: var(--sans-serif-font-regular);
    
    Look at lazy-styles.css for the font definitions and styles.css lines 47-52 for variables pointing to the fonts you need.

**Review by amol-anand — CHANGES_REQUESTED** [review]
everything else looks good. font change and you should be good to go.

File: styles/styles.css (hunk 1 above)
Comment (VladimirZaets) [reply styles/styles.css:660]: got it, will update it. I just copied the font name from figma

Comment (VladimirZaets) [reply styles/styles.css]: @amol-anand done

**Review by amol-anand — APPROVED** [review]

**Pull request merged**